bindings/node/native/src/tasks/tokenizer.rs

```diff
@@ -11,7 +11,7 @@ pub enum EncodeTask<'s> {
 }
 
 pub enum EncodeOutput {
-    Single(Encoding),
+    Single(Box<Encoding>),
```

Comment:
Is `Box` needed because Javascript? 😅

Reply:
No, clippy yells becuse that variant is 240 bytes long (pretty long for an enum). Setting it as a box just uses a pointer instead.